lib/internal/js_stream_socket.js

```diff
@@ -171,7 +171,7 @@ class JSStreamSocket extends Socket {
 
     this.stream.cork();
     // Use `var` over `let` for performance optimization.
-    for (var i = 0; i < bufs.length; ++i)
+    for (let i = 0; i < bufs.length; ++i)
```

Comment:
I would restrain from making this change unless we have ample evidence that it's as fast as.